### PR TITLE
Packages names updates

### PR DESCRIPTION
**Pull request by kocheick**
packages and names updated ~ https://github.com/maplibre/maplibre-plugins-android/pull/52

By the way, just pointing this out: I haven't touched / increased plugins individual version, since the main update has been package naming and no other feature added to the plugin itself.

### CODE REVIEW

File: plugin-annotation/src/test/java/org/maplibre/android/plugins/annotation/LineManagerTest.java [hunk 1]
```diff
@@ -2,28 +2,33 @@
...
+import org.maplibre.geojson.*;
 import org.maplibre.android.geometry.LatLng;
-import org.maplibre.android.maps.MapLibreMap;
 import org.maplibre.android.maps.MapView;
+import org.maplibre.android.maps.MapLibreMap;
 import org.maplibre.android.maps.Style;
 import org.maplibre.android.style.expressions.Expression;
-import org.maplibre.android.style.layers.LineLayer;
-import org.maplibre.android.style.layers.PropertyValue;
+import org.maplibre.android.style.layers.*;
```
Comment (louwers) [inline plugin-annotation/src/test/java/org/maplibre/android/plugins/annotation/LineManagerTest.java:14]: Please avoid glob imports.

**Comment by louwers** [comment]
Let me know when you can build everything locally so I can run CI.

File: plugin-offline/build.gradle [hunk 2]
```diff
@@ -19,7 +19,8 @@ android {
     }
 
     // Avoid naming conflicts
-    resourcePrefix 'mapbox_'
+//    resourcePrefix 'mapbox_'
```
Comment (louwers) [inline plugin-offline/build.gradle:22]: Can be removed.

**Comment by kocheick** [comment]
> Let me know when you can build everything locally so I can run CI.



> Let me know when you can build everything locally so I can run CI.

Let's gooooo

**Comment by louwers** [comment]
Testing tomorrow. Will likely merge tomorrow and make a release. Thanks.

**Comment by louwers** [comment]
I get this error when I try to run the sample app:

> Activity class {com.mapbox.mapboxsdk.plugins.testapp/org.maplibre.android.plugins.testapp.activity.FeatureOverviewActivity} does not exist

Tried cleaning the build.

**Review by louwers — APPROVED** [review]
Seems to work well!

**Pull request merged**